PatientWebApplication/HealthClinic.DAL/Service/PatientService.cs

```diff
@@ -54,18 +54,16 @@ public List<PatientUser> GetAll()
         }
 
 
-        public string ImageToSave(FileModel file)
+        public string ImageToSave(string path, FileModel file)
         {
             try
             {
-                string path = Directory.GetParent(Directory.GetCurrentDirectory()).FullName + "\\HealthClinic.DAL\\wwwroot\\" + file.FileName;
-
-
-                using (Stream stream = new FileStream(path, FileMode.Create))
+                string fullPath = path + "\\images\\" + file.FileName;
+                using (Stream stream = new FileStream(fullPath, FileMode.Create))
```

Comment:
The fullPath variable needs to be deleted and defined in the FileStream constructor.